csp-mixin/alerts/gcp-alerts.yml

```diff
@@ -14,3 +14,129 @@ groups:
           summary: 'CPU utilization is too high.'
           description: 'The VM {{ $labels.instance_name }} is under heavy load and may become unresponsive.'
           dashboard_uid: 'f115fe73641347c43415535d77e2dc0f'
+
+      - alert: GcpCEHighIOLatency
+        expr: |
+          avg by (job,project_id,instance_id)(stackdriver_gce_instance_compute_googleapis_com_instance_disk_average_io_latency{job=~".+",project_id=~".+",instance_id=~".+"}) > 5000
```

Comment:
Do you know the if the value is in seconds or in percentage? What means 5000? seconds?

Reply:
The value is in microseconds:

![image](https://github.com/user-attachments/assets/4e62b2db-e21a-45af-a6f0-4c302967221d)

Reply:
I think you can also add an alert for `instance/disk/average_io_queue_depth`